vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -308,6 +309,10 @@ void TextInputShadowNode::registerEvents() {
       });
 
   m_controlLoadedRevoker = control.Loaded(winrt::auto_revoke, [=](auto &&, auto &&) {
+    if (m_autoFocus) {
+      control.Focus(xaml::FocusState::Keyboard);
```

Comment:
Is there any defined behavior for what should happen when there are multiple components with autofocus prop enabled?

Reply:
Good question.

Reply:
Just tested on Windows and the last element to get Loaded will get focus, which is generally the last element in child order. I'll test on iOS and Android as well.

Reply:
Confirmed at least on iOS that the behavior is the same, the last TextInput in child order with autoFocus set to true will "win" focus.